src/Apps/Purchase/PurchaseApp.tsx

```diff
@@ -0,0 +1,41 @@
+import { PurchaseApp_orders } from "__generated__/PurchaseApp_orders.graphql"
+import { AppContainer } from "Apps/Components/AppContainer"
+import { SystemContext } from "Artsy"
+import { ErrorPage } from "Components/ErrorPage"
+import React, { useContext } from "react"
+import { Title } from "react-head"
+import { createFragmentContainer, graphql } from "react-relay"
+import { userIsAdmin } from "Utils/user"
+import { PurchaseHistoryFragmentContainer as PurchaseHistory } from "./Components/PurchaseHistory"
+
+export interface Props {
```

Comment:
```suggestion
export interface PurchaseAppProps {
```

It's a minor suggestion, but having this more explicit prevents accidental name collisions when doing `export * from 'blah'` in places. While technically it's very unlikely for this particular module to ever be referenced in that way, I still think it's somewhat of a best practice.